tshirts.cpp

```diff
@@ -20,3 +20,12 @@ void testTshirtSize() {
     assert(size(43) == 'L');
     std::cout << "All is well (maybe!)\n";
 }
+

```

Comment:
This file has both test code and production code. Consequence is that it will get delivered to all consumers of the functionality, resulting in code bloat.
Separate the test code, as you did in the previous assignment.